### DISCUSSION TITLE
Γραφικό περιβάλλον σε arch linux

### DISCUSSION THREAD

**Discussion by p20sami** [courses-ionio/hci#1621] [Announcements]
Καλησπέρα σας,
Εαν κατά την εγκατάσταση εγκαταστήσουμε κάποιο γραφικό περιβάλλον πχ kde, θα μπορούμε να έχουμε πρόσβαση κανονικά στο τερματικό, ή πρέπει να το αφήσουμε καθαρά τερματικό μόνο? Επίσης εάν εν τέλει έχουμε πρόσβαση στο τερματικό αφού εγκαταστήσουμε γραφικό περιβάλλον, θα έχει κάποια αρνητική βαθμολογική επίπτωση?
Σας ευχαριστώ εκ των προτέρων.

**Comment by epidrome** [comment]
σε τι διαφέρει με την συζήτηση εδώ #1595 

αυτό που σίγουρα βαθμολογούμε είναι η καλή-κακή χρήση της συζήτησης και των πόρων της ομάδας!

**Reply by p20sami** [reply]
Δεν είχα δει οτι υπήρχε σχετική συζήτηση, με συγχωρείτε.
